scripts/GetClrDbg.ps1

```diff
@@ -104,13 +104,14 @@ if (-not $RuntimeID) {
 }
 Write-Host "Info: Using Runtime ID '$RuntimeID'"
 
-# create the install folder if it does not exist
-if (-not (Test-Path -Path $InstallPath -PathType Container)) {
-    New-Item -ItemType Directory -Force -Path $InstallPath
+# create the temp folder if it does not exist
+$TempPath = Join-Path -Path $env:TEMP -ChildPath "getclrdbg"
+if (-not (Test-Path -Path $TempPath -PathType Container)) {
+    New-Item -ItemType Directory -Force -Path $TempPath
 }
```

Comment:
Should you delete the folder if it exists? I'm a little concerned that the file gets cached somewhere I don't know about and may not realize the need to clean to get an update.